README.md

```diff
@@ -43,24 +54,179 @@ The plugin will remove the custom attribute from the element and replace its con
+
+```
+
+```html
+<style>
+  .test {
+    text-transform: uppercase;
+  }
+
+  .test__hello {
+    color: red;
+  }
+
+  .test__world {
+    color: blue;
+  }
+</style>
+```
+
+#### Babel
+
+```html
+<script babel>
+  const hello = 'Hello World!'
+  let greeter = {
+    greet (msg) { alert (msg) }
+  }
+  greeter.greet(hello)
+</script>
+```
+
+```js
+const babel = require('babel-core')
+const options = { presets: ["es2015"], sourceMaps: false }
+
+const plugin = require('posthtml-content')({
+  babel: (js) => babel.transform(js, options).code
 })
 
 posthtml([plugin]).process(html)
 ```
 
 ```html
-<p>Wow, it's <strong>markdown</strong>!</p>
+<script>
+  'use strict';
+  var hello = "Hello World!";
+  var greeter = {
+    greet: function greet (msg) {
+      alert(msg);
+    };
+  };
+  greeter.greet(hello);
+</script>
 ```
 
-### License & Contributing
+## LICENSE & CONTRIBUTING
+- Details on running tests and contributing [can be found here](CONTRIBUTING.md)
+
+> MIT License
+
+> Copyright (c) 2016 static-dev
+
+> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+[npm]: https://img.shields.io/npm/v/posthtml-content.svg
+[npm-url]: https://npmjs.com/package/posthtml-content
+
+[node]: https://img.shields.io/node/v/gh-badges.svg
+[node-url]: https://nodejs.org
+
+[deps]: https://david-dm.org/static-dev/posthtml-content.svg
+[deps-url]: https://david-dm.org/static-dev/posthtml-content
+
+[devdeps]: https://david-dm.org/static-dev/posthtml-content/dev-status.svg
+[devdeps-url]: https://david-dm.org/static-dev/posthtml-content#info=devDependencies
+
+[style]: https://img.shields.io/badge/code%20style-standard-yellow.svg
+[style-url]: http://standardjs.com/
+
+[travis]: http://img.shields.io/travis/static-dev/posthtml-content.svg
+[travis-url]: https://travis-ci.org/static-dev/posthtml-content
+
+[travis-rel]: http://img.shields.io/travis/static-dev/posthtml-content.svg?branch=release/1.0.0
+[travis-rel-url]:https://travis-ci.org/static-dev/posthtml-content?branch=release/1.0.0
```

Comment:
At the moment there is none :) Indicates this modules is still in development and no first major version released. rm?